GiniVision/Classes/Core/Screens/Camera/CameraViewController.swift

```diff
@@ -329,10 +329,10 @@ extension CameraViewController {
     
     fileprivate func configurePopupViewForUnsupportedQR(_ newQRCodePopup: QRCodeDetectedPopupView,
                                                         dismissCompletion: @escaping () -> Void) {
-        newQRCodePopup.qrText.textColor = .red
+        newQRCodePopup.qrText.textColor = giniConfiguration.unsupportedQrCodePopupColor
         newQRCodePopup.qrText.text = .localized(resource: CameraStrings.unsupportedQrCodeDetectedPopupMessage)
         newQRCodePopup.proceedButton.setTitle("✕", for: .normal)
-        newQRCodePopup.proceedButton.setTitleColor(.red, for: .normal)
+        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupColor, for: .normal)
```

Comment:
Also then we should add an `unsupportedQrCodePopupButtonColor` as well to be analogous with `qrCodePopupButtonColor`. And also add the same highlighted state color as for the normal popup.
```suggestion
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor, for: .normal)
        newQRCodePopup.proceedButton.setTitleColor(giniConfiguration.unsupportedQrCodePopupButtonColor.withAlphaComponent(0.5),
                                                   for: .highlighted)
```